addons/account/models/account_move_line.py

```diff
@@ -765,10 +765,13 @@ def _compute_tax_tag_invert(self):
 
             elif origin_move_id.move_type == 'entry':
                 # For misc operations, cash basis entries and write-offs from the bank reconciliation widget
-                tax = record.tax_repartition_line_id.tax_id or record.tax_ids[:1]
-                is_refund = record.is_refund
-                tax_type = tax.type_tax_use
-                record.tax_tag_invert = (tax_type == 'purchase' and is_refund) or (tax_type == 'sale' and not is_refund)
+                if record.payment_id:
+                    record.tax_tag_invert = record.payment_id.payment_type == 'outbound'
```

Comment:
This should work the same as using `account.move.is_inbound()`, but without having to fetch the move the payment is linked to.
In my tests, credit notes had no epd line and receipts and bills had the same values for `tax_tag_invert`.

Reply:
it seems to work as expected, just awaiting confirmation from @tsb-odoo
I agree this would be easier, but keep in mind that 1 payment could be for many invoices/bills/credit notes etc
I'm not sure the comment above is entirely accurate now, perhaps that 1 should move and we should add another describing the payment condition